content/contribute/contributing-and-managing-patterns.adoc

```diff
@@ -0,0 +1,145 @@
+---
+menu: contribute
+title: Contributing and managing patterns
+weight: 50
+aliases: /contributing-and-managing-patterns/
+---
+
+:toc:
+:_content-type: ASSEMBLY
+include::modules/comm-attributes.adoc[]
+
+[id="contributing to patterns"]
+= Contributing to patterns
+
+To contribute, add the upstream repository as an additional remote and work on a branch other than main. This approach allows you to merge changes into the main branch (reflected in GitOps workflows) and makes upstream contributions easier. Contributions from your forked main branch typically include:
```

Comment:
consider rewriting to "To contribute, add the upstream repository as an additional remote and work on a branch other than the main branch. Merge changes into the main branch to align with GitOps workflows. This approach simplifies upstream contributions. Contributions from your forked main branch typically include:"  based on RedHat style recoomendation: "Do not use transitive verb      
                      constructions like 'allows                                   
                      you' to grant abilities to                                   
                      inanimate objects. Whenever                                  
                      possible, use direct,                                        
                      user-focused sentences where                                 
                      the subject of the sentence                                  
                      performs the action"